applications/OptimizationApplication/tests/algorithm_tests/analysis_based_tests/algorithm_steepest_descent/summary_orig.csv

```diff
@@ -0,0 +1,25 @@
+# Optimization problem ascii output
+# Kratos version: not_given
+# Timestamp     : not_specified
+# -----------------------------------------------
+# --------------- Initial values ----------------
+# 	algorithm:
+# 	mass_Structure:
+# 		initial_value: 1.386000000e+04
+# ------------ End of initial values ------------
+# -----------------------------------------------
+# ------------ Start of step values -------------
+# Headers:
+#  STEP, algorithm:std_obj_value, mass_Structure:value
```

Comment:
we do not we get % changes in rel and abs here ?!

Reply:
done